src/poker/game/GamePlay.java

```diff
@@ -0,0 +1,165 @@
+package poker.game;
+
+import poker.*;
+
+import java.util.*;
+
+public class GamePlay {
+    public static final Map<GamePhase, Integer> BOARD_PRINT_INDEXES = Map.of(
+            GamePhase.PREFLOP, 3,
+            GamePhase.FLOP, 4,
+            GamePhase.TURN, 5
+    );
+
+    public Deck deck;
+    public HandsAndBoard result;
+    public Player player1;
+    public Player player2;
+    GameState gameState;
+    List<Player> playerList;
+    public EquityEvaluator equityEvaluator;
+    public GameResult gameResult;
+    public ActionHandler actionHandler;
+
+    public GamePlay() {
+        this.deck = new Deck();
+        this.result = deck.shuffleDeckAndGetHandsAndBoard();
+        this.player1 = new Player(100, 0, 0, 0, result.getHand1(), "player1");
+        this.player2 = new Player(100, 0, 0, 0, result.getHand2(), "player2");
+        this.gameState = new GameState(GamePhase.PREFLOP, 0);
+        player1.setHandPlayer(result.getHand1());
+        player2.setHandPlayer(result.getHand2());
+        playerList = new ArrayList<>();
+        playerList.add(player1);
+        playerList.add(player2);
+        this.equityEvaluator = new EquityEvaluator(deck, result.getHand1(), result.getHand2(), result.getBoard());
+        this.gameResult = new GameResult();
+        this.actionHandler = new ActionHandler(gameState, result, deck, player1, player2, equityEvaluator);
+    }
+
+    public void gameStart() {
+        Player smallBlindPlayer = player2;
+        Player bigBlindPlayer = player1;
+        boolean outOfPostionPlayer = false;
+        while (playerList.size() > 1) {
+            Player nonActivePlayer = bigBlindPlayer;
+            Player activePlayer = smallBlindPlayer;
+            PossibleAction lastAction = null;
```

Comment:
jak zmienna nie musi byc tworzona w petli a zazwyczaj nie musi to wynies ja nad petle (kosmetycznie bedzie lepiej ze tworzenie zmiennych jest na samej gorze + wydaje mi sie ze java zmarcnuje troche czasu na "stworzenie" od nowa klasy)